tests/poster_mock.py

```diff
@@ -28,12 +28,7 @@ def start_server(port, **kwargs):
     """
     class PosterServerMock(BaseHTTPRequestHandler):
```

Comment:
@vladarefiev It's the base class for `PosterMocks`, each new class must extend it

Reply:
@paulodamaso As I can see every new class should extend your server `tests/integration/poster/poster_server_mock.py::PosterServerMock`, shouldn't it?

So why do we still need this one `tests/poster_mock.py::PosterServerMock`?

Reply:
@vladarefiev I'm sorry, I thoght that you was talking about 'tests/integration/poster/poster_server_mock.py::PosterServerMock'... We still need `tests/poster_mock.py::PosterServerMock` because all our tests are still dependent of it; when we refactor all the tests we could get rit of it